### PR TITLE
Spanner [GCP-7503]: Add documentation for enums

### PR DESCRIPTION
**Pull request by HemangChothani**
issue [7503]

### CODE REVIEW

File: spanner/docs/gapic/v1/admin_database_api.rst [hunk 1]
```diff
@@ -4,3 +4,6 @@ Spanner Admin Database Client API
 .. automodule:: google.cloud.spanner_admin_database_v1
   :members:
   :inherited-members:
+
+.. autoclass:: google.cloud.spanner_admin_database_v1.gapic.enums.Database
+    :members:
```
Comment (mf2199) [inline spanner/docs/gapic/v1/admin_database_api.rst]: This modifies GAPIC code which will have no effect as GAPICs are auto-generated from the protobufs.

File: spanner/docs/gapic/v1/admin_instance_api.rst [hunk 2]
```diff
@@ -4,3 +4,6 @@ Spanner Admin Instance Client API
 .. automodule:: google.cloud.spanner_admin_instance_v1
   :members:
   :inherited-members:
+
+.. autoclass:: google.cloud.spanner_admin_instance_v1.gapic.enums.Instance
+    :members:
```
Comment (mf2199) [inline spanner/docs/gapic/v1/admin_instance_api.rst]: Same here...

File: spanner/docs/gapic/v1/enums.rst [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+=============================
+
+.. automodule:: google.cloud.spanner_v1.gapic.enums
+    :members:
+
+.. autoclass:: google.cloud.spanner_v1.gapic.enums.ExecuteSqlRequest
+    :members:
+
+.. autoclass:: google.cloud.spanner_v1.gapic.enums.PlanNode
+    :members:
```
Comment (mf2199) [inline spanner/docs/gapic/v1/enums.rst]: ...and here...

File: spanner/docs/gapic/v1/admin_database_api.rst (hunk 1 above)
Comment (HemangChothani) [reply spanner/docs/gapic/v1/admin_database_api.rst]: @mf2199  Yes, GAPIC code is auto-generated but this gapic is in the docs folder and still unable to figure it out how they create a doc folder automatically . The files in the docs/gapic folder is responsible to create document of the classes which are defined in the .rst file.

**Comment by sumit-ql** [comment]
I think we cant do much on this issue as the code which generates these docs has some issue. Also @Emar-Kar and @mf2199 both saying correct that we should not modify docs manually as these are auto generated. We can ask more inputs from googlers in our weekly meeting.

**Pull request closed without merging**